po/pt-BR.po

```diff
@@ -4748,30 +4830,28 @@ msgstr ""
 "Como variáveis estáticas (`static`) são acessíveis de qualquer _thread_, "
 "elas precisam ser `Sync`. A mutabilidade interior é possível através de um "
 "[`Mutex`](https://doc.rust-lang.org/std/sync/struct.Mutex.html), atômico ou "
-"similar. Também é possível ter variáveis estáticas mutáveis, mas elas exigem "
-"sincronização manual de forma que qualquer acesso a elas requer código "
-"“inseguro”. Veremos [variáveis estáticas mutáveis](../unsafe/mutable-static-"
-"variables.md) no capítulo sobre _Unsafe Rust_."
+"similar."
 
 #: src/user-defined-types/static-and-const.md:82
-#, fuzzy
 msgid "Thread-local data can be created with the macro `std::thread_local`."
 msgstr ""
-"Dados de `thread_local` podem ser criados com a macro `std::thread_local`."
+"Dados locais da thread podem ser criados com a macro `std::thread_local`."
 
 #: src/user-defined-types/aliases.md:3
 msgid ""
 "A type alias creates a name for another type. The two types can be used "
 "interchangeably."
 msgstr ""
+"Um _alias_ de tipo cria um nome para outro tipo. Os dois tipos podem ser "
+"usados ​​de forma intercambiável."
 
 #: src/user-defined-types/aliases.md:13
 msgid "// Aliases are more useful with long, complex types:\n"
-msgstr ""
+msgstr "// _Aliases_ são mais úteis com tipos longos e complexos:\n"
```

Comment:
Should we be translating comments in the code snippets?

Reply:
I think so, because that's one of the changes with the mdbook, instead of adding the whole code snippet to the .po file, it's adding only the comments for potential localization.

Reply:
Yeah, _potential_ is the keyword: the translation system supports translation of comments and string literals in the code blocks. You should decide if you want to do this or not and document it somewhere for the language.